miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultPoolOrderService.java

```diff
@@ -140,6 +150,31 @@ protected void collectValidationErrors(PoolOrder object, PoolOrder beforeChange,
     if (!object.isDraft() && object.getOrderLibraryAliquots().isEmpty()) {
       errors.add(new ValidationError("Non-draft order must include at least one library aliquot"));
     }
+
+    if(strictPools) validateIndices(object, beforeChange, errors);
+  }
+
+  private void validateIndices(PoolOrder object, PoolOrder beforeChange, List<ValidationError> errors){
```

Comment:
What do you think about renaming this to something like `validateNoAdditionalDuplicateIndices`? (I'm imagining a future scenario where we might want to do additional validation on indices in all cases, not just in cases where `strictPools` is true)

Reply:
Done!